src/duplocloud/config.py

```diff
@@ -272,15 +272,44 @@ def cache_key_for(self, name: str):
       The cache key as a string.
     """
     h = self.host.split("://")[1]
-    k = f"{h},{name}"
+    parts = [h]
     if self.isadmin:
-      k = f"{k},admin"
-    return k
+      parts.append("admin")
+    parts.append(name)
+    return ",".join(parts)
+  
+  def expiration(self, hours: int = 6):
+    """Expiration
+    
+    Get the expiration time for the given number of hours. This is a simple calculation of the current time plus the number of hours.
+
+    Args:
+      hours: The number of seconds to add to the current time.
+
+    Returns:
+      The expiration time as a string.
+    """
+    return (datetime.datetime.now() + datetime.timedelta(hours=hours)).isoformat()
+  
+  def expired(self, exp: str = None):
```

Comment:
Yes and yes. I just have no idea how to at the moment. I went with 6hours as the most sane default I could find. Should getting and setting the correct expiration be another task or do you think this can be easily?